libfwupdplugin/README.md

```diff
@@ -1,12 +1,25 @@
-Planned API/ABI changes for next release
-========================================
-
- * Remove fu_common_is_cpu_intel()
- * Remove fu_firmware_strparse_uintXX()
- * Remove fu_plugin_get_usb_context() and fu_plugin_set_usb_context()
- * Remove fu_plugin_runner_usb_device_added(), fu_plugin_runner_udev_device_added()
-   and fu_plugin_runner_udev_device_changed()
- * Remove FuHidDevice->open() and FuHidDevice->close()
- * Remove FuUsbDevice->probe(), FuUsbDevice->open() and FuUsbDevice->close()
- * Remove FuUdevDevice->to_string(), FuUdevDevice->probe(), FuUdevDevice->open() and FuUdevDevice->close()
- * Remove fu_device_get_protocol() and fu_device_set_protocol()
+libfwupdplugin
+==============
+
+This library is deliberately API and ABI unstable. Think of it like the kernel,
+where keeping unused, unsafe and deprecated functions around forever is a
+maintainance burden.
```

Comment:
This is too strong of a statement.  I don't really want to see plugin API/ABI bumping every release or anything.  Can we meet somewhere in the middle?  We only bump API/ABI for plugins on major jumps?  IE fwupd 1.5.x to 1.6.?  Because otherwise we're going to have a harder time bringing this into LTS releases for the future.

Reply:
> We only bump API/ABI for plugins on major jumps?

I guess that's a compromise.

> we're going to have a harder time bringing this into LTS releases

Can you elaborate a bit please? For RHEL it would be acceptable as it's an internal interface, not designed to be used by other things.

Reply:
So then the next question becomes, if we branch for 1.6.0 now, at what point do we post-date the branch? e.g. do we want these things in 1.5.8?

* QUBES? [yes?]
* fu_common_write_uint16_safe[yes? seems safe and useful]
* battery_level [yes?]
* fu-ifd-firmware [no? on the logic that nothing uses it yet, and it might all change]
* fu_plugin_add_possible_quirk_key and the quirk prefix removal [yes? on the logic i want stuff easy to backport]
* FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID [yes? on the logic it fixes real bugs and has been tested really well...]

Reply:
Okay, I've pushed a 1_5_X branch now. I'll fix up master in a new PR now.

Reply:
I've softened the README -- does that look better?